packages/subgraph/src/utils/tokens/erc20.ts

```diff
@@ -95,6 +95,31 @@ export function fetchERC20(address: Address): ERC20Contract | null {
   return contract;
 }
 
+/**
+ * @dev Determines the type of ERC20 token (wrapped or regular) and returns its address.
+ *
+ * @param token The address of the token to be determined.
+ * @return tokenAddress The address of the ERC20 token if it's either wrapped or regular, null otherwise.
+ */
+export function determineERC20Token(token: Address): string | null {
```

Comment:
`determineERC20Token` might not be a telling name. How about `getUnderlyingERC20TokenAddress`?

Reply:
@Michael-A-Heuer  I dont think thats a good name either, the function checks if a token is wrapped or not and returns the instance id

Reply:
addressed in https://github.com/aragon/osx/pull/484/commits/945c0fb413f87f0ec138e5e03e7339367e1f9901